### PR TITLE
[Doc] add the fe configuration that controls hybrid table and update scenarios

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by decster — DISMISSED** [review]

File: docs/zh/administration/FE_configuration.md [hunk 1]
```diff
@@ -507,6 +507,12 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 - 默认值：TRUE
 - 引入版本：2.5.6
 
+#### enable_experimental_rowstore
+
+- 含义：是否开启行存混存特性（公测中）。设置为 `true` 表示开启，并且建表时在 `PROPERTIES` 中配置 `"STORE_TYPE" = "column_with_row"`，则可以实现数据同时按行和按列存储，可以在优化高并发点查和数据部分更新同时，也能保证列式存储的高效分析能力。更多使用说明，参见[行列混存表](../table_design/hybrid_table.md)。
```
Comment (amber-create) [inline docs/zh/administration/FE_configuration.md:512]: image
应该是“行列混存”

Comment (amber-create) [inline docs/zh/administration/FE_configuration.md:512]: 设置为 `true` 表示开启，并且建表时在 `PROPERTIES` 中配置 `"STORE_TYPE" = "column_with_row"`
改为 -> 设置为 `true` 开启该特性、并且建表时在 `PROPERTIES` 中配置 `"STORE_TYPE" = "column_with_row"`

File: docs/zh/table_design/hybrid_table.md [hunk 2]
```diff
@@ -2,43 +2,49 @@
...
-| 行列混存       | 数据同时按照列和行存储。简单来说，行列混存表会额外加一个隐藏的二进制类型的列 `__row`，写入数据至表的同时还会将一行数据所有 value 列编码后的值写入列 `__row`（如下所示）。由于数据同时按照列和行存储，因此会带来额外的存储成本。![img](../assets/table_design/hybrid_table.png) | <ul><li>适用于主键点查和部分列更新的场景，能大大降低随机 IO 和读写放大。</li><ul><li>点查（基于主键的简单查询，扫描和返回数据量小）。</li><li>表的字段个数比较少，并且通常会查询数据行的大部分字段或者整行数据的信息。</li><li>部分列更新。</li><li>行列混存表和预准备语句同时使用，能够通过节省解析 SQL 语句的开销从而提高查询性能，并且还能防止 SQL 注入攻击。</li></ul><li>同时也适用于复杂数据分析。</li></ul> |
-| 列存           | 按列存储数据。![img](../assets/table_design/columnar_table.png) | <ul><li>针对海量数据进行复杂或者即席查询。 </li><li>表的字段比较多（比如大宽表），但查询的字段不多。</li></ul> |
+| 行列混存       | 数据同时按照列和行存储。简单来说，行列混存表会额外加一个隐藏的二进制类型的列 `__row`，写入数据至表的同时还会将一行数据所有 value 列编码后的值写入列 `__row`（如下所示）。由于数据同时按照列和行存储，因此会带来额外的存储成本。![img](../assets/table_design/hybrid_table.png) | 兼顾行存和列存的场景，但是会带来额外的存储成本。<ul><li>按行存储的适用场景。</li><ul><li>基于主键的高并发点查。</li><li>表的字段个数比较少，并且通常会查询大部分字段。</li><li>部分列更新（更新多列和少量数据行）</li></ul><li>列存的场景：复杂数据分析。</li></ul> |
+| 列存           | 按列存储数据。![img](../assets/table_design/columnar_table.png) |复杂数据分析。 <ul><li>针对海量数据进行复杂查询分析，比如聚合分析，多表关联查询。 </li><li>表的字段比较多（比如大宽表），但查询的字段不多。</li></ul> |
```
Comment (amber-create) [inline docs/zh/table_design/hybrid_table.md:14]: image
改为顿号

File: docs/en/table_design/hybrid_table.md [hunk 3]
```diff
@@ -2,42 +2,49 @@
...
+| Hybrid row-column storage | Data is stored in both row-by-row and column-by-column fashions. Simply put, a table that uses hybrid row-column storage contains an additional, hidden binary-type column `__row`. When data is written to the table, all values from the value columns of a row are encoded and written to the `__row` column (as shown below). As the data is stored in both row-by-row and column-by-column fashions, additional storage costs are incurred. ![img](../assets/table_design/hybrid_table.png) | Suitable for use cases of both rowstore and columnstore storage xxx, but incurs additional storage costs.<ul><li>Use case of rowstore:</li><ul><li>High-concurrency point queries based on primary keys.</li><li>Queries against most fields from tables that consist of a small number of fields.</li><li>Partial column updates (more specifically, multiple columns and a small number of data rows need to be updated)</li></ul><li>Use case of columnstore: Complex data analysis.</li></ul>
```
Comment (amber-create) [inline docs/en/table_design/hybrid_table.md:13]: image
what "xxx" is supposed to be?

File: docs/en/table_design/hybrid_table.md [hunk 4]
```diff
@@ -2,42 +2,49 @@
...
+|
+| Column-oriented            | Data is stored in a column-by-column fashion ![img](../assets/table_design/hybrid_table.png) | Complex data analysis. <ul><li>Complex queries and analyses on massive datasets, such as aggregate analysis and multi-table join queries.</li><li>Tables consist of many fields (such as wide tables).  Queries involve only a few columns. </li></ul>|
```
Comment (amber-create) [inline docs/en/table_design/hybrid_table.md:15]: the rendering of this row is incorrect. You can see it using the Preview feature of GitHub.

Comment (amber-create) [inline docs/en/table_design/hybrid_table.md:15]: Tables consist of many fields (such as wide tables).  Queries involve only a few columns.
-> Tables consist of many fields (such as wide tables), but queries against these tables involve only a few columns.

File: docs/zh/table_design/hybrid_table.md [hunk 5]
```diff
@@ -2,43 +2,49 @@
...
 
 ## 基本操作
 
 ### 创建行列混存表
 
-建表时在 `PROPERTIES` 中配置 `"STORE_TYPE" = "column_with_row"`。
+1. 行列混存特性默认关闭，您可以设置 FE 动态参数 [`enable_experimental_rowstore`](../administration/FE_configuration.md#enable_experimental_rowstore) 为 `true` ，开启该特性。
```
Comment (amber-create) [inline docs/zh/table_design/hybrid_table.md:20]: image
多余空格

**Comment by hellolilyliuyi** [comment]
Important updates in this pr were already merge. See [41926](https://github.com/StarRocks/starrocks/pull/41926).

**Pull request closed without merging**